crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -344,6 +380,8 @@ impl<E: Backoff> State<E> {
         // Create sessions for all new nodes
         for added_node in added_nodes {
             for (provider_type, endpoints) in &self.local_endpoint_cache {
+                let paused = self.paused_peers.contains(&added_node); // Should be always
+                                                                      // false
```

Comment:
Is it considered an error state if it's not `false`?

Reply:
It should never be paused because it was just added. I removed the comment and added a warning instead :)